TLM/TLM/Manager/Impl/LaneConnectionManager.cs

```diff
@@ -168,7 +170,7 @@ public class LaneConnectionManager : AbstractGeometryObservingManager,
         /// <returns></returns>
         internal bool RemoveLaneConnection(uint laneId1, uint laneId2, bool startNode1) {
 #if DEBUGCONN
```

Comment:
Looks like an old define, change to `#if DEBUG` ? Multiple occurrences in file.

Reply:
`#if DEBUGCONN` replaced with `#if DEBUG`, but there is also `DEBUGSAVE`